### ISSUE TITLE
audio-/video streaming should have no download-button

### ISSUE THREAD

**Issue by berberisD** [nextcloud/server#10927]
### Steps to reproduce
1. share audio or video using the public link
2. open the link 

### Expected behaviour
sharing for "streaming" would be much better without download button (copyrights in some cases)
just like "youtube" - a button to start the stream would be enough
there should be an additional sharing-option like "only viewing online"

### Actual behaviour
you will see download-button in the middle of the site and a second one in the top

**Comment by berberisD** [comment]
I think the issues the nextcloud-bot found are not related to this case.
To clarify i have two screenshots here:
![screen2](https://user-images.githubusercontent.com/42811759/46025351-05156000-c0e9-11e8-89b1-5d6c5e982746.png)
So putting a tick at  hide-download-button
![screen1](https://user-images.githubusercontent.com/42811759/46025428-2c6c2d00-c0e9-11e8-93a5-daea28f785eb.png)
would deliver a page without explicit downloadbuttons.

Would be great, if this function could be installed :-)

**Comment by MorrisJobke** [comment]
cc @rullzer

**Comment by timmersr** [comment]
Checking "Hide download" will not only hide download options, but **also prevent video playback** in NC15 RC1.
![1](https://user-images.githubusercontent.com/31093459/49148137-ce391180-f306-11e8-8471-0ca47d321111.png)

**Comment by berberisD** [comment]
So the new "Hide download" in NC15 is planned to prevent the download-functionality in general.
-
Whereas  "Hide download buttons" is just to visually hide the button for downloads. 
With some tec skills its possible to download of course - but not so obvious anymore.

**Comment by MorrisJobke** [comment]
Fixed by #12531 and #11898

**Comment by MorrisJobke** [comment]
> Checking "Hide download" will not only hide download options, but **also prevent video playback** in NC15 RC1.

Separate issue.

**Comment by mattx388** [comment]
> > Checking "Hide download" will not only hide download options, but **also prevent video playback** in NC15 RC1.
> 
> Separate issue.

But this "Separate issue" will be fixed in the future ?

**Comment by Shishadampfer** [comment]
Hi all
I'm pretty new to nextcloud and the gallery app. Currently using the nextcloud version 16.0.5 with gallery app version 18.3.0. 
While testing I figured out that the download buttons still keep showing up, although I did check "hide download" for the specific share.  
While searching for a solution to remove the download option from the gallery I came across this thread. 

Any solution yet available or  planned to be implemented?
